blog/2024-06-14-june-releases.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "June 2024 Releases"
+slug: "2024-06-14-june-releases"
+authors: [dashagurova]
+tags:
+  [announcement, conda-index, conda-package-handling, conda-package-streaming]
+description: |
+  Conda-index 0.5.0, conda-package-handling 2.3.0 and conda-package-streaming 0.10.0 have been released! 🎉
+image: img/blog/2024-06-14-june-releases/banner.png
+---
+
+The June 2024 releases included updates to conda-index, conda-package-streaming and conda-package-handling! 🎉 All of these new releases have been released to both `main` and `conda-forge`.
```

Comment:
I was thinking it might be good to include links to the repos in case people aren't as familiar with these packages

```suggestion
The June 2024 releases included updates to [conda-index](https://github.com/conda/conda-index), [conda-package-streaming](https://github.com/conda/conda-package-streaming) and [conda-package-handling](https://github.com/conda/conda-package-handling)! 🎉 All of these new releases have been released to both `main` and `conda-forge`.
```